src/components/App/App.js

```diff
@@ -91,9 +91,11 @@ function App() {
         ) : null}
 
         <Container>
-          <Route
+          <RestrictedRoute
             path="/"
             exact
+            isAllowed={isLoggedIn(getToken())}
```

Comment:
Why are you duplicating these in every props instead of just hard coding them into the withRestricted access component? This is really confusing to reuse, if the user has to input props for the component to even work as expected

Reply:
Hardcoding it would make it unusable in any other scenario. I was planning on pulling this check out into a variable. Will update. 👍